src/frontend/komponenter/Fagsak/Behandlingsresultat/Behandlingsresultat.tsx

```diff
@@ -229,6 +266,24 @@ const Behandlingsresultat: React.FunctionComponent<IBehandlingsresultatProps> =
             {åpenBehandling.endretUtbetalingAndeler.length > 0 && (
                 <EndretUtbetalingAndelTabell åpenBehandling={åpenBehandling} />
             )}
+            {toggles[ToggleNavn.overgangsordningErTilgjengelig] &&
+                åpenBehandling.årsak == BehandlingÅrsak.OVERGANGSORDNING_2024 && (
+                    <>
+                        <OvergangsordningAndelTabell åpenBehandling={åpenBehandling} />
+                        {!erLesevisning && (
+                            <OvergangsordningAndel>
+                                <Button
+                                    variant="primary"
+                                    size="medium"
+                                    onClick={opprettOvergangsordningAndel}
+                                    icon={<StyledPlusIkon />}
+                                >
+                                    <Label>Legg til periode</Label>
+                                </Button>
```

Comment:
Er det meningen at knappen skal se annerledes ut enn knappen for `EndretUtbetalingsAndel`?